components/partials/swap/index.vue

```diff
@@ -0,0 +1,1574 @@
+<template>
+  <div class="bg-gray-800 rounded-xl py-4">
+    <div class="px-4 w-full">
+      <div class="flex items-center justify-center">
+        <v-button
+          class="text-center rounded-3xl focus:outline-none px-2 py-1 text-xs font-bold tracking-wide text-primary-500 hover:text-primary-600"
+          :class="{ 'text-gray-500': !tradingTypeMarket }"
+          @click.stop="onTradingTypeToggle(TradeExecutionType.Market)"
+        >
+          {{ $t('trade.market') }}
+        </v-button>
+        <div class="mx-2 w-px h-4 bg-gray-500"></div>
+        <v-button
+          class="text-center rounded-3xl focus:outline-none px-2 py-1 text-xs font-bold tracking-wide text-primary-500 hover:text-primary-600"
+          :class="{ 'text-gray-500': tradingTypeMarket }"
+          @click.stop="onTradingTypeToggle(TradeExecutionType.LimitFill)"
+        >
+          {{ $t('trade.limit') }}
+        </v-button>
+      </div>
+      <div class="mt-4">
+        <div class="bg-gray-950 pt-2 rounded-lg">
+          <div class="flex justify-between items-center mb-2 px-3">
+            <span class="text-2xs md:text-xs font-semibold uppercase tracking-wider text-gray-500 mt-auto text-center">
+              {{ $t('trade.swap.from') }}
+            </span>
+            <v-balance
+              v-if="market"
+              :balance="orderTypeBuy ? quoteAvailableBalance : baseAvailableBalance"
+              :market="market"
+            />
+          </div>
+          <v-token-selector
+            class="input-swap"
+            :amount="orderTypeBuy ? buyAmount : form.amount"
+            :balance="orderTypeBuy ? quoteAvailableBalance : baseAvailableBalance"
+            :value="fromToken"
+            :tokens="fromTokens"
+            :placeholder="'Select a token'"
+            @input:amount="onSetAmount"
+            @input:token="onSetFromToken"
+            @input:max="() => onMaxInput(100)"
```

Comment:
as described in https://github.com/InjectiveLabs/injective-dex/pull/676/files#r845947143

lets do this instead `@input:max=onMaxInput`